src/Plugin.InAppBilling.UWP/InAppBillingImplementation.cs

```diff
@@ -1,7 +1,10 @@
 using Plugin.InAppBilling.Abstractions;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
+using System.Xml;
+using Windows.ApplicationModel.Store;
 
 namespace Plugin.InAppBilling
```

Comment:
If you look in the project... essentially the same file is linked to older Windows Phone projects. this means the build will fail. You need to create a blank implementation for those projects.